extension/js/common/core/attachment.ts

```diff
@@ -199,10 +199,10 @@ export class Attachment {
       // ends with one of .gpg, .pgp, .???.asc, .????.asc
       return 'encryptedFile';
       // todo: after #4906 is done we should "decrypt" the encryptedFile here to see if it's a binary 'publicKey' (as in message 1869220e0c8f16dd)
+    } else if (this.name.match(/(cryptup|flowcrypt)-backup-([a-z0-9]+(?:\-[A-F0-9]{40})?)\.(key|asc)$/g)) {
```

Comment:
please move this private key check below public key check (as it was before this PR), as comment on line 201 is related to public keys parsing

Reply:
will do. I also updated the private key detection a bit.